bliss/surveys/survey.py

```diff
@@ -53,6 +59,14 @@ def predict_batch(self):
     def predict_batch(self):
         """Set a batch of data for prediction."""
 
+    def nmgy_to_nelec(self):
```

Comment:
A lot of surveys (e.g., DC2) don't use `nanomaggies` as their units of flux. Is there a more generic way to encode the calibration parameters?

Reply:
calibration = conversion from counts to physical units (but not necessarily nanomaggies)

Reply:
Yes certainly; a more generic function name and/or arguments should suffice.

Reply:
Done.